### ISSUE TITLE
Keras RNN example for bAbI dataset, can not reproduce accuracy results?

### ISSUE THREAD

**Issue by stefanm91** [keras-team/keras#8061]
I tried the babi_rnn.py located in keras/examples and in the comment section of the same file it says it achieves 100% accuracy on task 1 (one supporting fact) with 1000 training samples in 20 epochs.

I did not change any of the hyperparameters and the test accuracy is around 50% (only_supporting = False). Could please let me know whether you get the same results and if so what could be the problem?

I tried changing also changing the hyperparameters by increasing the embedding size to 200 and number of epochs to 40. It increases the train accuracy to 94% but test accuracy is around 72% with the GRU model.

I am running newest version keras + tensorflow

Thank you.

**Comment by bdwyer2** [comment]
I can reproduce this issue. Keras 2.0.8 / TensorFlow 1.3.0 /  Linux Mint / 1080 ti

The claim was made at 22c091ae3ff1048c0a6d28edca7c57bb93378850
